charts/springboot/templates/cronjobs.yaml

```diff
@@ -1,6 +1,6 @@
 {{ range $name, $definition := .Values.cronJobs }}
 ---
-apiVersion: "batch/v1beta1"
+apiVersion: "batch/v1"
```

Comment:
Consider using (Capabilities)[https://helm.sh/docs/chart_template_guide/builtin_objects/]:

```apiVersion: "{{- if Capabilities.APIVersions.Has "batch/v1" }} "batch/v1" {{- else -}} "batch/v1beta1" {{- end }}"```

This way you'd support old version as well.

N.B. untested code, check whitespace and adjust `-` inside `{{ }}` accordingly.

Reply:
Good idea 👍, updated

Reply:
LGTM